src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -726,7 +726,7 @@ private Voucher withRequiredFields(Voucher voucher, Invoice invoice) {
            String expenseClassExtAccountNo = fundDistrs.getKey();
            FundExtNoExpenseClassExtNoPair key = new FundExtNoExpenseClassExtNoPair(fundExternalAccountNo, expenseClassExtAccountNo);
            List<FundDistribution> fundDistributions = fundDistrs.getValue();
-           fundDistributions.forEach(fundDistribution -> fundDistribution.setCode(fund.getCode()));
+           fundDistributions.forEach(fundDistribution -> fundDistribution.setCode(fund.getCode() + "-" + fundDistribution.getCode()));
```

Comment:
`fundDistribution.getCode()` may return **null** at that stage, so final code will be `fundCode-null`